oci-ggs/create/create-gb.md

```diff
@@ -275,5 +59,4 @@ You may now **proceed to the next lab.**
 ## Acknowledgements
 * **Author** - Jenny Chan, Consulting User Assistance Developer, Database User Assistance
 * **Contributors** -  Denis Gray, Database Product Management; Katherine Wardhana, User Assistance Developer
-* **Last Updated By/Date** - Katherine Wardhana, January 2024
-
+* **Last Updated By/Date** - Katherine Wardhana, JaFebruarynuary 2024
```

Comment:
@kwardhana Typo with Last Updated By/Date

Reply:
@dannymgithub sorry about that, updated!

Reply:
@kwardhana never an issue, I was just unsure of what month to put